### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AshleyHenry15**
### Description

 Lechlan (Support) proposed some changes to the documentation for clarity and to provide additional examples.

Please see the [Slack](https://rstudio.slack.com/archives/CUC92M4TT/p1626399469181500) thread for details

Connected to #

### Testing Notes / Validation Steps

1. Pulled/forked master
2. Made changes to `README`
3. Ran `make build`
4. Previewed changes to `index.md` file:

### CODE REVIEW

**Review by mmarchetti — APPROVED** [review]

**Comment by AshleyHenry15** [comment]
@mmarchetti I can't merge this since I don't have write access so can someone merge this when it is ready? Thnx!

**Pull request merged**
